example/server.rb

```diff
@@ -50,7 +50,11 @@
   conn = HTTP2::Server.new
   conn.on(:frame) do |bytes|
     # puts "Writing bytes: #{bytes.unpack("H*").first}"
-    sock.write bytes
+    if sock.is_a?(TCPSocket)
+      sock.send bytes, 0
```

Comment:
It's a little weird to use send in one and write in another. I believe send is just an alias.. can we use write? Related, do we actually need the if check here?